common-theurgy.lic

```diff
@@ -38,7 +38,7 @@ module DRCTH
   ] unless defined?(DEVOTION_LEVELS)
 
   def has_holy_water?(theurgy_supply_container, water_holder)
-    DRCI.get_item?(water_holder, theurgy_supply_container)
+    return unless DRCI.get_item?(water_holder, theurgy_supply_container)
```

Comment:
Should this be `return false unless ...` since the function is a predicate?

Reply:
I swear I am just making working for you, even if this is an amazing learning opportunity for me... Thank you!

Reply:
I'm happy to help :)